### DISCUSSION TITLE
Integration plugins for Obsidian / LogSeq / Markdown [Apr 20]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5557] [Discord Threads]
> 🙋‍♂️ **Hioko**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### Currently no. What's your use case? Are you referring to support one of the column as rich text (markdown?) column? Or ability to import contents onto NocoDB?  
#  
> 🙋‍♂️ **Hioko**  
#### obsidian/markdown editors are good (quick) for non-tabular/non-structured data, nocodb is good at strucutred data/tables. so their marriage will make world happy  
#  
> 🙋‍♂️ **Hioko**  
#### importing/exporting content between nocodb & markdown editors  
#  
> 💁 **o1lab**  
#### can u pls expand on the usecase of import/export of nocodb to markdown editors?  
#  
> 🙋‍♂️ **Hioko**  
#### there could be several, but currently if i say if exam subject- result table is stored in nocodb, is there any way to access it(access it or refer/link to it) from obsidian query/notes?  
#  
> 🙋‍♂️ **Hioko**  
#### i can draft many other use cases  
#  
> 💁 **o1lab**  
#### ahok we arent aware of obsidian query   
#### if it can access nocodb apis (or plugin within their ecosystem) - it should be doable  
#  
> 🙋‍♂️ **Hioko**  
#### or viceversa?  
#  
> 🙋‍♂️ **Hioko**  
#### e.g. if i have written this sentence in logseq/obsidian "GPT-4 is wonderful AI program"  
#  
> 🙋‍♂️ **Hioko**  
#### is it possible to access/query/link from nocodb?  
#  
> 🙋‍♂️ **Hioko**  
#### so that if i make any table related to AI tools in nocodb i can put that link  
#  
> 🙋‍♂️ **Hioko**  
#### nocodb should also marry with omnivore  
#  
> 💁 **o1lab**  
#### not possible to query from nocodb at the moment  
#  
> 🙋‍♂️ **Hioko**  
#### so i guess this is missing link between structred & non-structured tools, currenlty  
#  
> 🙋‍♂️ **Hioko**  
#### need to bridge that gap. of-course, markdown supports table, but it is not as advanced/user-friendly as spreadsheet/dedicated tabular tools like nocodb/excel/google-sheet  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1087410608728703098)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
